modules/pairIdSystem.js

```diff
@@ -35,11 +38,6 @@ export const pairIdSubmodule = {
    * @type {string}
    */
   name: MODULE_NAME,
-  /**
-   * used to specify vendor id
-   * @type {number}
-   */
-  gvlid: 755,
```

Comment:
Make this configurable. Should be the DSP

Reply:
this is not included in the bid request at all. this is the owner of the module. we need prebid to confirm. 
the prefered thing to do is to leave as-is.

Reply:
use `VENDORLESS_GVLID` here